frontend/src/components/maps/MinePin.js

```diff
@@ -58,7 +58,7 @@ export class MinePin extends Component {
           this.setState({ isFullMap: false });
         } else {
           this.setState({ isFullMap: true });
-          mineIds = this.props.mineIds;
+          mineIds = [...this.props.mineIds];
```

Comment:
Does something break if this change is reverted? It looks like this should be logically equivalent to `mineIds = this.props.mineIds`

Reply:
This was a linting error :P